reactive/swarm.py

```diff
@@ -24,14 +38,15 @@ def swarm_etcd_cluster_setup(etcd):
     leading with the agent, connecting to the discovery service, then follow
     up with the manager container on the leader node.
     """
-    bind_docker_daemon()
     con_string = etcd.connection_string().replace('http', 'etcd')
+    bind_docker_daemon(con_string)
 
     opts = {}
     opts['connection_string'] = con_string
     opts['addr'] = hookenv.unit_private_ip()
-    opts['port'] = 2375
+    opts['port'] = 2376
```

Comment:
Further down in the code you are using 3376 and 2375. I am confused at why 2376 is correct here.